setup.py

```diff
@@ -32,6 +30,6 @@
      , version = version
      , zip_safe = False
      , package_data = {'aspen': ['request_processor/mime.types']}
-     , install_requires = ASPEN_DEPS
-     , tests_require = TEST_DEPS
+     , install_requires = open('requirements.txt').read()
+     , tests_require = open('tests/requirements.txt').read()
```

Comment:
What's the effect of this on `sdist`s of this package? Do we need to include the requirements files in `MANIFEST.in` or whatevs?